en/getting-started/dashboard/index.md

```diff
@@ -26,7 +26,7 @@ Recently opened projects can be quickly accessed through the **Project** tab. Th
 - **2** - Other versions of the editor can be selected to open the project.
 - **3** - Sort projects by time in forward/reverse order (new in version 1.0.12).
 - **4** - Search projects directly by project name (new in version 1.0.12).
-- **Add** button - For importing other projects.
+- **Add** button - For importing other projects. Since version **v1.0.13**, allows dragging and droping project into the project list.
```

Comment:
```suggestion
- **Add** button - For importing other projects. Since **v1.0.13**, allows dragging and droping project into the project list directly.
```